app/views/surveys/component/_form-footer.pug

```diff
@@ -3,10 +3,20 @@ section.form-footer
     if !headless
       .form-footer__content__main-controls  
         if type === 'preview'
+          if nextPrompt
+            fieldset.draft.question.next-prompt
+              .option-wrapper: label 
+                input.ignore(type='checkbox', name='next-prompt') 
+                span #{nextPrompt}
           button#validate-form.btn.btn-primary
             i.icon.icon-check= ' '
             span(data-i18n='formfooter.validate.btn')= t('formfooter.validate.btn')
         else if type === 'fieldsubmission' || type === 'view'
+          if nextPrompt && completeButton
+            fieldset.draft.question.next-prompt
+              .option-wrapper: label 
+                input.ignore(type='checkbox', name='next-prompt') 
+                span #{nextPrompt}
```

Comment:
might be good to give the span the 'option-label' class (like in the form) so the styling moves along with any changes to checkbox form styling (in the future). However, if you're engaged in an annoying CSS fight when doing this, don't bother.

Reply:
sure style updated